src/main/resources/config/liquibase/tables_postgres.sql

```diff
@@ -0,0 +1,187 @@
+-- Thanks to Patrick Lightbody for submitting this...
+--
+-- In your Quartz properties file, you'll need to set 
+-- org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
```

Comment:
where is this file used?

Reply:
it's meant for a re-creation of Quartz DB tables - I have added info to the README.